python/xorbits/_mars/dataframe/reduction/tests/test_reduction_execution.py

```diff
@@ -1061,3 +1061,185 @@ def test_custom_series_aggregate(setup, check_ref_counts):
 
     result = s.agg(MockReduction2())
     assert pytest.approx(result.execute().fetch()) == data.agg(MockReduction2())
+
+
+@require_cudf
+@require_cupy
+@pytest.mark.parametrize(
+    "method",
+    [
+        "all",
+        "any",
+        "count",
+        "cummax",
+        "cummin",
+        "cumprod",
+        "cumsum",
+        "kurtosis",
+        "max",
+        "min",
+        "nunique",
+        "prod",
+        "size",
+        "sem",
+        "skew",
+        "std",
+        "sum",
+        "unique",
+        "var",
+    ],
+)
+def test_gpu_series_agg(method, setup_gpu):
```

Comment:
Is it possible to parameterize the input? So that `test_gpu_series_agg` and `test_gpu_dataframe_agg` can be merged into one.

Reply:
> Is it possible to parameterize the input? So that `test_gpu_series_agg` and `test_gpu_dataframe_agg` can be merged into one.

I tied to merge these two UTs into one. However, for series and df, infer func is ndependent, so the specific exception types thrown by some operators are also different. If I want to show the specific in UT, I still need to specifically judge whether the series or df, and the code is not reduced.